cockatrice/src/tab_deck_editor.cpp

```diff
@@ -336,7 +336,8 @@ void TabDeckEditor::retranslateUi()
     aLoadDeck->setText(tr("&Load deck..."));
     aSaveDeck->setText(tr("&Save deck"));
     aSaveDeckAs->setText(tr("Save deck &as..."));
-    aLoadDeckFromClipboard->setText(tr("Load deck from cl&ipboard..."));
+    aLoadDeckFromClipboard->setText(tr("Load deck from clipboard..."));
+    aLoadDeckFromClipboard->setShortcut(QKeySequence("Ctrl+L"));
```

Comment:
Using <kbd>Ctrl + V</kbd> combination caused issues on Mac OSX and "L" for "Load" seems more realistic to me. I'm fine changing this back if that's what you'd like.